### ISSUE TITLE
Undefined error

### ISSUE THREAD

**Issue by Anticept** [EDCD/coriolis#250]
Playing around with a data gathering fit ([see this fit](https://coriolis.edcd.io/outfit/diamondback_explorer?code=A0pataFflddfsaf5---0e02--483202v62i2f.AwRj4zKA.CwRgDBld4iQ%3D.H4sIAAAAAAAAA2P8Z8%2FAwPCXCUhwvGBmYOCX%2BPP%2Fv5gmOwPDf%2BZ%2FUiApFiAhEsAPlAr4%2Ff%2B%2F0BFOBgaJDUDB%2F6z%2FLEHyQE0Mf5qABGcNUCsXSEoRxFICsVRBLDWwcrZ%2FiSDlrEBCSMGPgUFvwov%2F%2FyU7hIFS3AgpPY8v%2F%2F9LgEQlVbWBUv8ZQAAAWJZE2qYAAAA%3D.EweloBhBmUA4QFMCGBzANokICMF9A%3D%3D%3D)) I came across this when clicking the wrench icon for the frame shift wake scanner as pictured.

![image](https://user-images.githubusercontent.com/521038/39082856-d7aa0586-4527-11e8-8c23-78c070371fbc.png)

```
Jameson, we have a problem..
TypeError: b.Modifications.modules[i.grp].blueprints[i.blueprint.fdname] is undefined


Please note that this site uses Google Analytics to track performance and usage. If you are blocking cookies, for example using Ghostery, please disable blocking for this site and try again.

Create an issue on Github if this keeps happening. Add these details:
Browser: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:59.0) Gecko/20100101 Firefox/59.0
Path: /outfit/diamondback_explorer?code=A0pataFflddfsaf5---0e02--483202v62i2f.AwRj4zKA.CwRgDBld4iQ%3D.H4sIAAAAAAAAA2P8Z8%2FAwPCXCUhwvGBmYOCX%2BPP%2Fv5gmOwPDf%2BZ%2FUiApFiAhEsAPlAr4%2Ff%2B%2F0BFOBgaJDUDB%2F6z%2FLEHyQE0Mf5qABGcNUCsXSEoRxFICsVRBLDWwcrZ%2FiSDlrEBCSMGPgUFvwov%2F%2FyU7hIFS3AgpPY8v%2F%2F9LgEQlVbWBUv8ZQAAAWJZE2qYAAAA%3D.EweloBhBmUA4QFMCGBzANokICMF9A%3D%3D%3D
Error: Unknown
Details:

scriptUrl: https://coriolis.edcd.io/app.92dfb5.js
line: 31
col: 348516
error: {}
```

**Comment by willyb321** [comment]
This is an issue with the guard I added to prevent crashes when importing, will fix today or tomorrow.

**Comment by willyb321** [comment]
Temp fixed in 157c1148fb0b49b78325efde165722493c5359f2

**Comment by willyb321** [comment]
Releasing now
